Docs - Use the main branch for the Wiki (#104)
iotaledger/IOTA-2.0-Research-Specifications: 6.4 Finalization.md

---

> description: As every node might have slightly different perceptions of the Tangle at a given time, so a notion of Finality is necessary to guarantee consensus on the Tangle and its ledger state.

Reviewer: ```suggestion
description: Since every node might have a slightly different perception of the Tangle at a given time, a notion of Finality is necessary to guarantee consensus on the Tangle and its ledger state.
```
